docs/devel/owners.md

```diff
@@ -0,0 +1,72 @@
+# Owners files
+
+_Note_: This is a design for a feature that is not yet implemented.
+
+## Overview
+We want to establish owners for different parts of the code in the Kubernetes codebase.  These owners
+will serve as the approvers for code to be submitted to these parts of the repository.  Notable, owners
+are not necessarily expected to do the first code review for all commits to these areas, but they are
+required to approve changes before they can be merged.
+
+## High Level flow
+
+### Step One: A PR is submitted
+After a PR is submitted, a user can request the approval from the required owners.  The user requests this
```

Comment:
There were some complains about the bot spamming PRs (and thus email) trying to balance things here, I'm ok w/ auto sending on every PR also...